crmd/lrm.c

```diff
@@ -2317,6 +2317,26 @@ do_lrm_event(long long action,
     CRM_CHECK(FALSE, return);
 }
 
+static char *
+remove_escaped_newlines(const char *string)
+{
+    char *pch = NULL;
+    char *ret = NULL;
+    static const char *escaped_newline = "\\n";
+
+    if (!string) {
+        return NULL;
+    }
+
+    ret = strdup(string);
+
+    while ((pch = strstr(ret, escaped_newline)) != NULL) {
+        memmove(pch, pch + strlen(escaped_newline), strlen(pch + strlen(escaped_newline)) + 1);
+    }
+
+    return ret;
+}
```

Comment:
Newlines gets escaped when metadata is stored in op->output. I thought it could prevent metadata conversion to xmlNode, but it actually works all right, so we don't need this function.